modules/bigtable-instance/main.tf

```diff
@@ -25,30 +24,45 @@ locals {
       }
     ]
   ])
+
+  clusters_autoscaling = flatten([
```

Comment:
it's better if you preserve the map type from the variable, as `for_each` on lists/sets uses positional indexes as keys, and it's very very easy to inadvertently drop multiple resources if an item is added or removed

Reply:
I am still learning the ropes with Terraform, I have changed this type back to map, thanks for the tip, I was not sure about how to do it or whether it could be done. I will be submitting a new commit soon.

Reply:
Commit added, resolving conversation to enable auto merge.